pkg/services/provisioning/provisioning.go

```diff
@@ -18,34 +18,43 @@ import (
 )
 
 type ProvisioningService interface {
+	RunInitProvisioners() error
```

Comment:
Is it ok to add this to the interface since I need it in the override. (I can always type assert to registry.Service in the override but this is cleaner IMO)

Reply:
Probably fine.

Reply:
Thanks :pray: